src/index.ts

```diff
@@ -0,0 +1,6 @@
+import express from 'express';
+import * as routes from './routes';
```

Comment:
Hm, I feel like the other side of the no-default-export rule is that we use the exported names directly! Maybe instead:

```suggestion
import { api } from './routes';
```

(and likewise replace `routes.api` with `api` below)

Reply:
My first thought was "hmm, `api` isn't a great variable name since out of the context of routes it isn't clear what it is" and then my second thought was "that should probably be `apiRoutes` in the export anyway!"

Sounds good and I agree.

Reply:
Ah, perfect! Yeah I stumbled on that, too, but didn't get as far as you did - I like that solution!